.github/workflows/deploy.yml

```diff
@@ -63,6 +63,8 @@ jobs:
 
       - name: Deploy on master branch
         if: github.ref == 'refs/heads/master'
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
How is `GITHUB_TOKEN` then used by the script?

Reply:
The aim to give to ci action write access by enable write permission on GITHUB_TOKEN scope (in settings)